string-manipulation-1.tar/string-manipulation-1/src/is-upper-cased.js

```diff
@@ -0,0 +1,25 @@
+/* exported isUpperCased */
+/**
+ * PSEUDOCODE:
+ *
+ * Define a function, named `isUpperCased`, that accepts one argument, `word`.
+ *   Within the function, implent an if else statment, that checks if every character in the word is
+ * uppercased. The result should be one of two boolean values, true or false.
+ * Start by using an if statement, by using the if keyword and opening a parenthesis that checks if the
+ * argument word, is strictly equal to word with the string method, toUpperCase() applied to the argument.
+ * toUpperCase(), converts all the letters to uppercased, within the string.
+ * If the word argument, contains all caps in the string, then the value should return true.
+ * Use the else statement, and open curly bracket and close it. Within, the brackets return fasle and that
+ * would be the answer if the string contains any lowercased letters.
```

Comment:
This is good but it's a little overly complicated. Try to keep one line of pseudocode to match one line of your function. Also if you can, try to avoid using specific code language... for example you could say something like this:

Define a function isUpperCased that takes one parameter, `word`
check to see if the `word` has all upper cased characters
if it does, return true
otherwise, return false.